schemas/search.yml

```diff
@@ -0,0 +1,81 @@
+---
+- name: search
+  title: Search
+  group: 2
+  short: Fields to describe details of a search event.
+  description: >
+    The Search fields describe information about a search request event: query
+    or pagination. The fields that should be used with this field set include:
+    `event.action` to describe the search action (`search.query`,
```

Comment:
Let's make it a bit clearer that it's not mandatory here, by simply saying that these are examples

```suggestion
    `event.action` to describe the search action (e.g. `search.query`,
```